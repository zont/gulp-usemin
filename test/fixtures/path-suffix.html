<!-- build:css style.css -->
<link rel="stylesheet" href="css/clear.css"/>
<link rel="stylesheet" href="css/main.css"/>
<!-- endbuild -->

<!-- build:js app.js -->
<script src="js/lib.js"></script>
<script src="js/main.js"></script>
<!-- endbuild -->