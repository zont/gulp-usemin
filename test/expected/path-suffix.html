<link rel="stylesheet" href="style.css?t=1440587840169"/>

<script src="app.js?t=1440587840169"></script>